src/Language/Haskell/Liquid/Types/PrettyPrint.hs

```diff
@@ -492,7 +492,6 @@ filterReportErrorsWith FilterReportErrorsArgs {..} errs =
     then continue
     else do
       case reducedErrs of
-        -- XXX(matt.walker) there is a law that can simplify this, probably
```

Comment:
Don't know how this got into this commit, but I decided to remove this comment since it was more of a WIP thing when I made it.